lib/seahorn/BvOpSem2.cc

```diff
@@ -3182,6 +3186,45 @@ Optional<APInt> Bv2OpSem::agg(Type *aggTy,
   return res;
 }
 
+/// Crab stuff
+// Build Crab CFG, run crab analyses, etc.
+void Bv2OpSem::mkCrabAnalysis(const llvm::Module &M,
+                              seadsa::GlobalAnalysis &dsa,
+                              TargetLibraryInfoWrapperPass &tli) {
+
+  std::unique_ptr<clam::HeapAbstraction> heap_abs(
```

Comment:
do not allocate on the heap. That is, don't use `new`

Reply:
maybe I'm wrong on that, lets check with @caballa

Reply:
`heap_abs` is passed to 
  ```
  CrabBuilderManager(CrabBuilderParams params,
                                     llvm::TargetLibraryInfoWrapperPass &tli,
                                     std::unique_ptr<HeapAbstraction> mem);
```

So that heap allocation is forced by the Clam API.